packages/wells/src/client/model/Survey.ts

```diff
@@ -0,0 +1,42 @@
+import { SequenceRow } from 'stable/dist/src/api/sequences/sequenceRow';
+
+export type SearchSurveys = (args?: any) => Promise<Survey[]>;
+
+export interface Survey {
+  /**
+   * @type {number}
+   * @memberof Survey
+   */
+  id: number;
+  /**
+   * @type {string}
+   * @memberof Survey
+   */
+  name: string;
+  /**
+   * @type {string}
+   * @memberof Survey
+   */
+  description: string;
+  /**
+   * @type {number}
+   * @memberof Survey
+   */
+  assetId: number;
+  /**
+   * @type {string}
+   * @memberof Survey
+   */
+  externalId?: string;
+  /**
+   * Custom, application specific metadata. String key -> String value.
+   * @type {{ [key: string]: string; }}
+   * @memberof Survey
+   */
+  metadata?: { [key: string]: string };
+  /**
+   * @type {Promise<SequenceRow[]>}
+   * @memberof Survey
+   */
+  rows(limit?: number): Promise<SequenceRow[]>;
```

Comment:
See comment above (below?) - I believe this should probably be a method instead of an attribute, but I may be wrong due to not knowing the language enough :)

Reply:
it is declared as a method :+1: : 
![image](https://user-images.githubusercontent.com/25263842/98250350-9bc6d300-1f77-11eb-8e33-5a265ce0b3be.png)